vite.config.ts

```diff
@@ -45,6 +38,8 @@ const config: UserConfig = {
     },
   },
   define: {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    USE_HASH_ROUTING: true,
```

Comment:
Shouldn't it be false?

Reply:
yeah should be false in this config 👍  still polishing

Reply:
okay has been resolved by [f03b97b](https://github.com/radicle-dev/radicle-interface/pull/478/commits/f03b97baf22707a4c7a3ac2a36f6a427c54e3b04)